### PR TITLE
Add symbols count to status bar. Initial status values fix

### PR DESCRIPTION
**Pull request by vladson**
I needed a feature to show full symbols count of the field (valuable if some length validation is present on the backend). Also, when editor is initialized on not empty fields, values of statuses where wrong, I could not help but fixed it to compute on initialization also.

### CODE REVIEW

**Comment by WesCossick** [comment]
By symbols, do you mean number of characters? If so, can you change the word "symbols" to "characters" so that it's more intuitive. Also, can you set up a jsFiddle that demonstrates this change?

**Comment by vladson** [comment]
Oh, sure, "characters" is more appropriate status label. js fiddle with example is here. https://jsfiddle.net/uLwsL7t9/1/

File: src/js/simplemde.js [hunk 1]
```diff
@@ -900,7 +900,7 @@ function SimpleMDE(options) {
 
 	// Handle status bar
 	if(!options.hasOwnProperty("status")) {
-		options.status = ["autosave", "lines", "words", "cursor"];
+		options.status = ["autosave", "lines", "words", "characters", "cursor"];
```
Comment (WesCossick) [inline src/js/simplemde.js]: Can you remove `characters` from the default behavior? I like the default having just the other four.

**Comment by WesCossick** [comment]
After that, I'll merge it in. Thanks for contributing to SimpleMDE.

**Comment by vladson** [comment]
Ok, I removed 'characters' from defaults.

**Comment by WesCossick** [comment]
Sorry about this, but one final change. Per the [contributing guide](https://github.com/NextStepWebs/simplemde-markdown-editor/blob/master/CONTRIBUTING.md), I'll need you to compare against the `development` branch. You'll need to submit another PR to do that.

**Pull request closed without merging**